platform/darwin/src/MGLStyle.mm

```diff
@@ -77,7 +77,6 @@ @interface MGLStyle()
 @property (nonatomic, readonly, weak) MGLMapView *mapView;
 @property (nonatomic, readonly) mbgl::style::Style *rawStyle;
 @property (readonly, copy, nullable) NSURL *URL;
-@property (nonatomic, readwrite, strong) NS_MUTABLE_DICTIONARY_OF(NSString *, MGLOpenGLStyleLayer *) *openGLLayers;
```

Comment:
Looks like this dictionary wound up unused after #8626. The original purpose of this dictionary was to allow `-layerWithIdentifier:` to return a fully initialized MGLOpenGLStyleLayer object that still had its callback implementations intact. As long as that’s still working correctly, this change looks good.

Reply:
@1ec5 verified

Reply:
@1ec5 `[MGLStlye layerFromMBGLLayer]` already takes care of this for all layer types.